### PR TITLE
added  Loading animation in studio

### PR DESCRIPTION
**Pull request by riyanrathore26**
feat: added animation in studio before showing error message
issue: #915 
Before adding animation 

https://github.com/asyncapi/studio/assets/97107462/afc71ca8-467d-4b65-8dd1-2d2207eb6c60

After add animation

https://github.com/asyncapi/studio/assets/97107462/9e70cea0-1eb1-4e2b-9f28-e0c568d575b8

### CODE REVIEW

File: apps/studio/src/components/Navigation.tsx [hunk 1]
```diff
@@ -252,6 +252,7 @@ export const Navigation: React.FunctionComponent<NavigationProps> = ({
   className = '',
 }) => {
   const [hash, setHash] = useState(window.location.hash);
+  const [loading,setloading] = useState(false);
```
Comment (Amzani) [inline apps/studio/src/components/Navigation.tsx:255]:
```suggestion
  const [loading, setloading] = useState(false);
```

**Review by Amzani — CHANGES_REQUESTED** [review]
@riyanrathore26 could you fix linting errors as well

**Comment by riyanrathore26** [comment]
> @riyanrathore26 could you fix linting errors as well

@Amzani Can you explain in more brief?

**Review by Amzani — APPROVED** [review]

**Review by satishkumarsajjan — COMMENTED** [review]
Here setTimeout is being used to create that loading state feel. Which does not solve the mentioned issue. A new approach should be used to solve the issue.

**Pull request merged**